genesis/engine/solvers/rigid/rigid_solver_decomp.py

```diff
@@ -4537,7 +4535,7 @@ def get_links_acc_ang(self, links_idx=None, envs_idx=None, *, unsafe=False):
         tensor = ti_field_to_torch(self.links_state.cacc_ang, envs_idx, links_idx, transpose=True, unsafe=unsafe)
         return tensor.squeeze(0) if self.n_envs == 0 else tensor
 
-    def get_links_COM(self, links_idx=None, envs_idx=None, *, unsafe=False):
```

Comment:
Maybe we should add an API for real links_COM? In a next PR

Reply:
Yes, would be great adding this in the next PR :)